keps/sig-storage/2848-transfer-volumesnapshots-namespaces/kep.yaml

```diff
@@ -0,0 +1,56 @@
+title: Transfer VolumeSnapshot Namespaces
+kep-number: 2848
+authors:
+  - "@elbehery"
+  - "@huffmanca"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
+status: provisional
+creation-date: 2021-08-02
+reviewers:
+  - "@xing-yang"
+  - "@mhenriks"
+  - "@jsafrane"
+  - "@mkimuram"
+  - "@bswartz"
+approvers:
+  - "@xing-yang"
+  - "@jsafrane"
+  - "@saad-ali"
+
+##### WARNING !!! ######
+# prr-approvers has been moved to its own location
+# You should create your own in keps/prod-readiness
+# Please make a copy of keps/prod-readiness/template/nnnn.yaml
+# to keps/prod-readiness/sig-xxxxx/00000.yaml (replace with kep number)
+#prr-approvers:
+
+see-also:
+  - "https://github.com/kubernetes/enhancements/pull/2326"
+  - "https://github.com/kubernetes/enhancements/pull/2110"
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.23"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.23"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: VolumeSnapshotNamespaceTransfer
+      components:
+        - kube-apiserver (assuming we introduce the StorageTransfer* resources)
```

Comment:
Remove kube-apiserver and clarify that this is an out-of-tree feature gate for external-snapshotter

Reply:
+1 🙏🏽